configs/otel-collector-config.yaml

```diff
@@ -0,0 +1,23 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+exporters:
+  logging:
+    loglevel: debug
+  prometheus:
```

Comment:
i added a prometheus exporter config here and then put it under the `service - pipeliens - metrics - exporters` key below (next to `logging`)